cli/ops/net.rs

```diff
@@ -30,14 +43,14 @@ pub fn init(i: &mut Isolate, s: &State) {
 #[derive(Deserialize)]
 struct AcceptArgs {
   rid: i32,
+  transport: String,
```

Comment:
Rather than adding this `transport` arg, we could (potentially) check the type in the resource table using https://doc.rust-lang.org/std/any/struct.TypeId.html and branch based on if the listener rid is a TCP or Unix resource.

Reply:
Good idea, will look into this!